ci-operator/step-registry/stackrox/e2e/stackrox-e2e-ref.yaml

```diff
@@ -21,6 +21,8 @@ ref:
     default: ''
   - name: QA_TEST_TARGET
     default: ''
+  - name: COLLECTION_METHOD
+    default: core_bpf
```

Comment:
one thing to consider is that the step is shared with existing release CI. Maybe this should default to "ebpf" until all outstanding releases that might not support "core_bpf" reach end of support. The master, release and nightly configurations can explicitly choose the collection method they want.

Reply:
Good point.